### PR TITLE
Print error if remotefile had nodeData since before

### PR DESCRIPTION
**Pull request by sander-visser**

### CODE REVIEW

File: apx/common/src/apx_nodeManager.c [hunk 1]
```diff
@@ -543,9 +543,13 @@ static void apx_nodeManager_createNode(apx_nodeManager_t *self, const uint8_t *d
                         {
                            APX_LOG_ERROR("[APX_NODE_MANAGER] createOutInitDataFromDefinition failed");
                         }
-                        apx_nodeData_setNodeInfo(nodeData, nodeInfo);
-                        apx_fileManager_sendFileOpen(fileManager, outDataFile->fileInfo.address);
                      }
+                     else
```
Comment (cogu) [inline apx/common/src/apx_nodeManager.c:549]: I have a hard time understanding the consequence of this change. Can we please meet up in person to discuss?

File: apx/common/src/apx_nodeManager.c [hunk 2]
```diff
@@ -543,9 +543,13 @@ static void apx_nodeManager_createNode(apx_nodeManager_t *self, const uint8_t *d
                         {
                            APX_LOG_ERROR("[APX_NODE_MANAGER] createOutInitDataFromDefinition failed");
                         }
-                        apx_nodeData_setNodeInfo(nodeData, nodeInfo);
+			apx_nodeData_setNodeInfo(outDataFile->nodeData, nodeInfo);
```
Comment (cogu) [inline apx/common/src/apx_nodeManager.c]: You're using tabs on this line only. Please change to spaces.

**Review by cogu — CHANGES_REQUESTED** [review]

File: apx/common/src/apx_nodeManager.c (hunk 2 above)
Comment (sander-visser) [reply apx/common/src/apx_nodeManager.c]: Argh - did a quick vi edit - sorry

**Comment by sander-visser** [comment]
Elipse apx_unit_test passed (OK (68 tests)

**Review by cogu — APPROVED** [review]

**Pull request merged**